.travis.yml

```diff
@@ -2,14 +2,22 @@ language: go
 
 python: 2.7
 
+services:
+  - mysql
+
 before_install:
   - pip install --upgrade pip virtualenv
   - virtualenv "${HOME}/FSENV"
+  - mysql --print-defaults
 
 install:
   - source "${HOME}/FSENV/bin/activate" && pip install -e .
   - go get -v -t ./... # Install dependencies needed for tests.
 
+before_script:
+  - export MYSQL_TEST_USER=travis
+  - export MYSQL_TEST_ADDR=127.0.0.1:3306
```

Comment:
Use the 'env' attribute to set env variables e.g. see https://github.com/google/grr/blob/master/.travis.yml

Reply:
As discussed offline, outside of a matrix, this seems to create an implicit matrix. Adding an explicit matrix seems to change other things. Leaving as is.

Reply:
SGTM. We will likely want an explicit matrix when we add testing for other platforms, but we can think about that later.